src/Orc.Controls/Controls/StepBar/Views/StepBar.xaml.cs

```diff
@@ -0,0 +1,62 @@
+﻿namespace Orc.Controls
+{
+    using System.Collections.Generic;
+    using System.Windows;
+    using System.Windows.Controls;
+
+    /// <summary>
+    /// Interaction logic for StepBar.xaml
+    /// </summary>
+    public sealed partial class StepBar
+    {
+        #region Constructors
+        public StepBar()
+        {
+            Loaded += OnLoaded;
```

Comment:
remove, all Catel views already have an OnLoaded method that can be overridden and take care of / prevent double loading, etc.